### DISCUSSION TITLE
希望openai的文本长度可以自定义

### DISCUSSION THREAD

**Discussion by chaniqure** [YiiGuxing/TranslationPlugin#5521] [Ideas]
目前的openai的翻译的文本长度太短了，大多数源码的注释都没办法一键翻译，直接就超出范围了。。。希望可以自定义openai的翻译文本长度

**Comment by YiiGuxing** [comment]
在 3.6 版本会增加这个长度限制的，增加到了10000字符，大多数情况应该够用了，太长会被截断的。
